src/Manavo/BootstrapForms/FormBuilder.php

```diff
@@ -203,13 +202,14 @@ protected function checkable($type, $name, $value, $checked, $options)
     public function checkbox(
         $name,
         $value = 1,
-        $label = null,
         $checked = null,
         $options = []
     ) {
         $checkable = parent::checkbox($name, $value, $checked, $options);
 
-        return $this->wrapCheckable($label, 'checkbox', $checkable);
+        return @$options['label'] ?
```

Comment:
Could you change this to not be `@$options['label']`, but to be `!empty($options['label'])`. Better to check rather than suppress the potential error!